addons/udes_get_info/tests/common.py

```diff
@@ -113,13 +100,25 @@ def create_move(cls, picking, **kwargs):
         Move = cls.env["stock.move"]
         unit = cls.env.ref("uom.product_uom_unit")
 
-        vals = {
-            "picking_id": picking.id,
-            "product_uom": unit.id,
-            **kwargs,
-        }
+        vals = {"picking_id": picking.id, "product_uom": unit.id, **kwargs}
         return Move.create(vals)
 
+    def complete_picking(self, picking, dest_location, **picking_kwargs):
```

Comment:
This in udes_get_info and the commit says udes_stock, is this an error?

Reply:
yeah lol - forgot to remove it